cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/DatasetServiceClient.java

```diff
@@ -271,13 +271,12 @@ private HttpResponse doDelete(String resource) throws DatasetManagementException
     return doRequest(HttpMethod.DELETE, resource);
   }
 
-  private HttpResponse doRequest(HttpMethod method, String resource,
-                                 @Nullable Multimap<String, String> headers,
-                                 @Nullable String body) throws DatasetManagementException {
+  private HttpResponse doRequest(HttpMethod method, String resource, String body,
```

Comment:
why reordering? Doesn't it make more sense to keep them in the the order in which they come in the actual HTTP request?